addons/l10n_ec/data/account_tax_report_data.xml

```diff
@@ -576,6 +612,44 @@
                             </record>
                         </field>
                     </record>
+                    <record id="tax_report_line_parent_line_report_62" model="account.report.line">
+                        <field name="name">Acquisitions and local payments (excludes fixed assets) taxed at 5% (with the right to tax credits)</field>
+                        <field name="children_ids">
+                            <record id="tax_report_line_104_540" model="account.report.line">
+                                <field name="name">Gross value(540)</field>
+                                <field name="code">c540</field>
```

Comment:
Same here :wink:

Reply:
The same reason that @pepetreshere wrote in the previous comment